src/memo/memo.mli

```diff
@@ -101,7 +101,6 @@ module Make_map_traversals (Map : Map.S) : sig
   val parallel_iter : 'a Map.t -> f:(Map.key -> 'a -> unit t) -> unit t
   val parallel_map : 'a Map.t -> f:(Map.key -> 'a -> 'b t) -> 'b Map.t t
 end
-[@@inline always]
```

Comment:
callout: `@inline` annotations don't belong in interfaces. they are a property of the implementation: https://github.com/ocaml/ocaml/issues/12972#issuecomment-1938287079

Reply:
prior work: https://github.com/ocaml-dune/fiber/pull/25